calculator/main.py

```diff
@@ -13,7 +13,13 @@ def create_new_calculator(operations=None):
     :param operations: Dict with initial operations.
                        ie: {'sum': sum_function, ...}
     """
-    pass
+    calculator = {'operations':{}, 'history':[]}
+    if operations is not None:
+        if not isinstance(operations, dict):
+            raise InvalidParams()
+        for key, value in operations.items():
```

Comment:
Instead of iterating through and manually setting each key/value pair you can use `.update()` to merge a dictionary into another one. Example:
```
my_dict = {'a': 1, 'b': 2, 'c': 3}
my_other_dict = {'d': 4, 'e': 5}
my_dict.update(my_other_dict)
```
Now `my_dict` will be: `{'a': 1, 'b': 2, 'c': 3, 'd': 4, 'e': 5}`